js/components/ladderPage/ladder.tsx

```diff
@@ -49,43 +56,29 @@ export const Ladders = ({ routeId }: { routeId: RouteId }): ReactElement => {
   );
 
   return (
-    <>
-      <span>
-        Status:{" "}
-        {vehiclePositions !== null && tripUpdates !== null ?
-          <>
-            Connected ({vehiclePositions.length} vehicles, {tripUpdates.length}{" "}
-            tripUpdates)
-          </>
-        : <>Loading...</>}
-      </span>
-
-      {/* outer <div> prevents horizontal overflow from affecting the overall page layout
-      inner <div> enables horizontal overflow (scrolling) for the 3 StationLists
-      ^ this should potentially be handled in a future <LadderPage />. */}
-      <div className="overflow-x-hidden">
-        <div className="relative flex xl:justify-center overflow-x-auto snap-x snap-mandatory">
-          {Array.from(vehiclesByBranch.entries()).map(
-            ([stationList, vehicles], index) => (
-              <TrainsAndStations
-                key={index}
-                ladderConfig={stationList}
-                vehicles={vehicles}
-              />
-            ),
-          )}
-        </div>
-      </div>
-    </>
+    <div className="relative flex flex-grow min-[1200px]:justify-center overflow-x-auto snap-x snap-mandatory">
```

Comment:
Not sure I understand why, but removing `justify-center` seemingly helps with the sidebar overlap problem?

Reply:
It's thresholded, and needed because we center-align at big sizes and left-align on mobile 👍 check out HEAD, it's okay now I think